vscode-dotty/src/worksheet.ts

```diff
@@ -183,21 +199,18 @@ class Worksheet implements Disposable {
       this.runVersion += 1
     }
 
-    return vscode.workspace.applyEdit(addNewLinesEdit).then(_ => {
-      for (let line of resultLines) {
-        const decorationPosition = new vscode.Position(actualLine, 0)
-        const decorationMargin = this.margin - editor.document.lineAt(actualLine).text.length
-        const decorationType = this.createDecoration(decorationMargin, line)
-        const decorationOptions = { range: new vscode.Range(decorationPosition, decorationPosition), hoverMessage: line }
-        const decoration = new Decoration(decorationType, decorationOptions)
-
-        this.decoratedLines.add(actualLine)
-        this.decorations.push(decoration)
-
-        editor.setDecorations(decorationType, [decorationOptions])
-        actualLine += 1
-      }
-    })
+    const _ = await vscode.workspace.applyEdit(addNewLinesEdit);
```

Comment:
this will actually create a value named `_`, I think you can just do `await vscode.workspace.applyEdit(addNewLinesEdit)`